README.md

```diff
@@ -142,3 +142,17 @@ It does this by defaulting to the bundle provided by `golang.org/x/crypto/x509ro
 See https://github.com/fortio/multicurl for a good example.
 
 If you do not want this behavior, build using `-tags no_tls_fallback`.
+
+### Small binaries
+
+If you application doesn't do http request/response logging at all, use `-tags no_http` or `-tags no_net` (that one will also imply `no_tls_fallback`) and save 1.6 Mbytes in your binary's size (because the linker isn't smart enough to see net/http isn't actually used) (1.1Mbytes for log's no_http and 530k from the no_tls_fallback)
```

Comment:
Nonblocking:

For constituency I would use, a space plus sane units


```suggestion
If you application doesn't do http request/response logging at all, use `-tags no_http` or `-tags no_net` (that one will also imply `no_tls_fallback`) and save 1.6 Mbytes in your binary's size (because the linker isn't smart enough to see net/http isn't actually used) (1.1 Mbytes for log's no_http and 530 kbytes from the no_tls_fallback)
```

or

```suggestion
If you application doesn't do http request/response logging at all, use `-tags no_http` or `-tags no_net` (that one will also imply `no_tls_fallback`) and save 1.6 MB in your binary's size (because the linker isn't smart enough to see net/http isn't actually used) (1.1 MB for log's no_http and 530 kB from the no_tls_fallback)
```